tests/integration/test_update_analysis.py

```diff
@@ -0,0 +1,81 @@
+import json
+from flask.testing import FlaskClient
+from http import HTTPStatus
+
+from trailblazer.constants import TrailblazerStatus
+from trailblazer.server.schemas import AnalysisUpdateRequest
+from trailblazer.store.models import Analysis
+
+TYPE_JSON = "application/json"
```

Comment:
Feels general / move it to constants

Reply:
It is only used in the tests, so I'll leave it as a constant in this test file